metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020-Present Okta Inc. -->
+<component type="console-application">
+    <id>com.okta.developer.CLI</id>
+    <name>Okta CLI</name>
+    <developer_name>Okta Inc.</developer_name>
+    <project_license>Apache-2.0</project_license>
+    <metadata_license>CC0-1.0</metadata_license>
+    <summary>The Okta CLI is the easiest way to get started with Okta!</summary>
+    <description>
+        <p>The Okta CLI helps developers get started securing applications using Okta.</p>
+    </description>
+    <url type="homepage">https://github.com/oktadeveloper/okta-cli</url>
+    <url type="bugtracker">https://github.com/oktadeveloper/okta-cli/issues</url>
+    <provides>
+        <binary>okta</binary>
+    </provides>
+
+    <content_rating type="oars-1.1" />
+    <icon type="remote" height="128" width="128">
+        https://raw.githubusercontent.com/flathub/com.okta.developer.CLI/master/icons/okta-logo-128.png
```

Comment:
@nedrichards
This seems to be what others packages are doing for icons,  (it looks like there is a requirement for a "remote" icon)
The current URL is `https://raw.githubusercontent.com/bdemers/flathub/com.okta.developer.CLI/icons/okta-logo-128.png` but I'm assuming if the package is accepted it would be `flathub/com.okta.developer.CLI`

Reply:
It's a bug (probably by design though) of appdata tooling used by flatpak-builder, which assumes that `console-application` does not need or ship an icon. For now I'd just add `"skip-icons-check": true` to flathub.json.

Reply:
done!